doc/HOWTO_oracle.md

```diff
@@ -57,92 +76,79 @@ sudo ln -s libocci.dylib.12.1   libocci.dylib
 sudo ln -s /opt/oracle/instantclient/sqlplus /usr/local/bin/sqlplus
 ```
 
-##### Add Environment Variables to profile
+### Set Up Environment Variables
 
-* Add to `~/.profile` (bash) or `~/.zprofile` (zsh)
+1. Add to `~/.profile` (bash) or `~/.zprofile` (zsh)
 
 ```
-export NLS_LANG="AMERICAN_AMERICA.AL32UTF8"
-export ORACLE_HOME=/opt/oracle/instantclient
-export OCI_DIR=$ORACLE_HOME
+# The Oracle adapter for ActiveRecord uses this password to connect as a
+# system user, to be able to create and drop databases appropriately.
+export ORACLE_SYSTEM_PASSWORD=Oradoc_db1
+
+# This is used to specify the default language and encoding for Oracle clients
+export NLS_LANG="AMERICAN_AMERICA.UTF8"
 ```
 
-* `source ~/.profile` or `source ~/.zprofile`, respectively
+1. `source ~/.profile` or `source ~/.zprofile`, to load the changes you just made
 
 
-##### Test your installation:
+### Test Your Installation
 
-* Run this command:
+To connect to the Oracle server, run:
 
 ```
-sqlplus myuser/mypassword@//myserver:1521/mydatabase.mydomain.com
+sqlplus "sys/Oradoc_db1@(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=localhost)(PORT=1521))(CONNECT_DATA=(SERVER=DEDICATED)(SERVICE_NAME=ORCLCDB.localdomain)))" as sysdba
 ```
 
-* Verify you see an output like this:
+Your output should show something like this:
 
 ```
-SQL*Plus: Release 12.1.0.2.0 Production on Fri Mar 3 13:48:51 2017
+Connected to:
+Oracle Database 12c Enterprise Edition Release 12.2.0.1.0 - 64bit Production
 
-Copyright (c) 1982, 2016, Oracle.  All rights reserved.
-
-ERROR:
-ORA-12154: TNS:could not resolve the connect identifier specified
-
-
-Enter user-name:
+SQL> 
 ```
 
-* `^D` out of that and carry on.
-
 ## Setting up the Database
 
-##### Install Required Gems
-
-* Make sure you have the `activerecord-oracle_enhanced-adapter` and `ruby-oci8` gems
-enabled.
-
-##### Set up database configuration
-
-* Copy from oracle settings template
-
-```
-cp config/database.yml.oracle.template config/database.yml
-```
-
-* Validate keys in settings are correct for your current application
-
-* Start, setup, and seed the oracle container:
-
-```
-docker/oracle/setup.sh
-# demo:seed is optional
-rake demo:seed
-```
+1. Make sure you have the `activerecord-oracle_enhanced-adapter` and `ruby-oci8` gems enabled in your `Gemfile`.
+1. Copy the Oracle template for `database.yml` by running `cp config/database.yml.oracle.template config/database.yml`.
+1. Run `bundle exec rake db:setup`.
```

Comment:
When I got to this step, it asked me for the oracle system password. We should probably note that in the instructions.

Reply:
Did you set the env variable as mentioned on line 86, which talks about this?

Reply:
Ah, makes sense. That feels like kind of an odd thing to me to add into your dotfiles, though.

Reply:
You don’t have to—you can specify it when you invoke `rake db:setup` or `rake db:drop` or any other similar tasks, but I found it’s easiest to put this in a dotfile, kinda like `NLS_LANG`.